projects/1.19/assets/fastfood-delight/fastfooddelight/lang/zh_cn.json

```
@@ -0,0 +1,38 @@
+{
+  "block.fastfooddelight.checkout_machine": "收银机",
+
+  "item.fastfooddelight.fries_bowl": "炸物袋",
+  "item.fastfooddelight.cheese": "芝士",
+
+  "item.fastfooddelight.french_fries": "薯条",
+  "item.fastfooddelight.carrot_fries": "炸胡萝卜条",
+  "item.fastfooddelight.fried_onions": "炸洋葱圈",
+
+  "item.fastfooddelight.cod_sandwich": "鳕鱼三明治",
+  "item.fastfooddelight.salmon_sandwich": "鲑鱼三明治",
+  "item.fastfooddelight.pork_sandwich": "猪肉三明治",
+  "item.fastfooddelight.cheeseburger": "芝士堡",
+  "item.fastfooddelight.chicken_cheese_sandwich": "芝士鸡腿堡",
+  "item.fastfooddelight.croque_monsieur": "芝士火腿三明治",

```

```suggestion
  "item.fastfooddelight.croque_monsieur": "法式三明治",
```
```suggestion
  "item.fastfooddelight.croque_monsieur": "法式男士三明治",
```
模组里没有croque madame所以感觉男士也并不必要